images/ceph/Dockerfile

```diff
@@ -20,7 +20,11 @@ ARG TINI_VERSION
 
 # Run tini as PID 1 and avoid signal handling issues
 RUN curl --fail -sSL -o /tini https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini-${ARCH} && \
-    chmod +x /tini
+    chmod +x /tini && \
+    curl -LJO https://github.com/ceph/persistent-volume-migrator/releases/download/v0.1.0-alpha/pv-migrator && \
+    chmod +x pv-migrator && \
```

Comment:
You don't need `chmod` since there is the `install` command below.

Reply:
removed. Thanks!